### DISCUSSION TITLE
Documentation for VRT APIs

### DISCUSSION THREAD

**Discussion by alexniculae** [Visual-Regression-Tracker/Visual-Regression-Tracker#355] [Q&A] [answered]
Hello,

Is there any documentation around the VRT APIs and how the results of a run can be accessed?

I tried and had a look over the code, but unfortunately I couldn't figure this one out...

Thank you 🙂

**Comment by pashidlos** [comment, marked as answer]
sure, there is swagger docs that you could review 
you could access it via the backend url by adding `/api`
like here https://visual-regression-tracker.com:4200/api/
